go/common/ledger/ledger.go

```diff
@@ -0,0 +1,35 @@
+// Packaged ledger contains the common constants and functions related to Ledger devices
+package ledger
+
+import (
+	ledger "github.com/zondax/ledger-oasis-go"
+)
+
+const (
+	// ListingPathPurpose is set to 44 to indicate use of the BIP-0044 specification.
+	ListingPathPurpose uint32 = 44
+	// ListingPathCoinType is set to 118, the number owned by Cosmos via SLIP-0044 registration.
+	// TODO: Update this number after SLIP-0044 registration is complete.
+	ListingPathCoinType uint32 = 118
+	// ListingPathAccount is the account index used to list and connect to Ledger devices by address.
+	ListingPathAccount uint32 = 0
+	// ListingPathChange indicates an external chain.
+	ListingPathChange uint32 = 0
+	// ListingPathIndex is the address index used to list and connect to Ledger devices by address.
+	ListingPathIndex uint32 = 0
+)
+
+var (
+	// ListingDerivationPath is the path used to list and connect to devices by address.
+	ListingDerivationPath = []uint32{ListingPathPurpose, ListingPathCoinType, ListingPathAccount, ListingPathChange, ListingPathIndex}
+)
+
+type LedgerDevice = ledger.LedgerOasis
+
+func ListDevices() {
+	ledger.ListOasisDevices(ListingDerivationPath)
```

Comment:
This command requires user confirmation for all devices listed, which is pretty annoying and unnecessary from a user's perspective. @jleni is there a chance that we can add a function to `ledger-oasis-go` that gets the address and does not require user confirmation?

Reply:
Sorry, that should have been exposed there! It is actually part of the API and it is already exposed in JS. Adding that right now to the go library.